infrastructure/aws/trigger_blame_rebuild.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import boto3
+from datetime import datetime, timedelta
+import sys
+import os.path
+
+# Usage: trigger_blame_rebuild.py <mozsearch-repo> <config-repo> <config-input> <branch> <channel>
+#  e.g.: trigger_blame_rebuild.py https://github.com/mozsearch/mozsearch https://github.com/mozsearch/mozsearch-mozilla config1.json master release
+
+def trigger(mozsearch_repo, config_repo, config_input, branch, channel):
+    ec2 = boto3.resource('ec2')
+    client = boto3.client('ec2')
+
+    running = ec2.instances.filter(Filters=[{'Name': 'tag-key', 'Values': ['blame-builder']},
+                                           {'Name': 'tag:channel', 'Values': [channel]},
+                                           {'Name': 'instance-state-name', 'Values': ['running']}])
+    for instance in running:
+        print("Terminating existing running indexer %s for channel %s" % (instance.instance_id, channel))
+        instance.terminate()
+
+    user_data = '''#!/usr/bin/env bash
+
+cd ~ubuntu
+sudo -i -u ubuntu ./update.sh "{branch}" "{mozsearch_repo}" "{config_repo}"
+sudo -i -u ubuntu mozsearch/infrastructure/aws/main.sh rebuild-blame.sh 168 "{branch}" "{channel}" config "{config_input}"
```

Comment:
The 168 is less intuitive here than the 10 for the indexer.  I think it would be great to have a comment in the script here or have the value extracted out into a constant with a comment.  In particular, I think it would be great for the comment to capture how close we are to 168 hours at the time of writing and what date that was written.  (I realize there may be the caveat that maybe you're just doing this locally and haven't run on EC2?)

Reply:
Ah, good point. And yeah I haven't actually done a full run on EC2 yet so I don't know how long it will actually take. So for now maybe I'll land it with the 168 extracted into a variable, and then after I do a full run I can write a better comment.